schemas/emergency_contact.py

```diff
@@ -0,0 +1,32 @@
+from ma import ma
+from models.emergency_contact import EmergencyContactModel
+from schemas.contact_number import ContactNumberSchema
+from utils.time import time_format
+from marshmallow import fields, validates, ValidationError
+
+
+class EmergencyContactSchema(ma.SQLAlchemyAutoSchema):
+    class Meta:
+        model = EmergencyContactModel
+
+    created_at = fields.DateTime(time_format)
+    updated_at = fields.DateTime(time_format)
+
+    contact_numbers = fields.List(fields.Nested("ContactNumberSchema"))
+
+    @validates("contact_numbers")
+    def validate_contact_nums(self, value):
```

Comment:
Excellent. At first glance I thought there was a bug in this logic, but after checking it out. It works fine! - The only thing missing are some validation tests in the schema test file. 
Since I already wrote a test to check my curiousity. Here is what I used:
```
    def test_validate_contact_number(self, empty_test_db):
        invalid_payload = {
            'name': 'emergency contact name',
            'contact_numbers': [{}]
        }
        validation_errors = EmergencyContactSchema().validate(invalid_payload)

        assert 'contact_numbers' in validation_errors
```

I thought we would need a raise statement here, but I guess not.

Reply:
Ah I see there are some integration tests for this, however those could be removed in favor of the unit tests. I'd like to see unit tests created for these new validation methods, however we could do it in a separate ticket too.

Reply:
So this method should be tested within the EmergencyContact schema tests?

Reply:
Yes please